### PR TITLE
Send events when list item has loaded.

### PR DESCRIPTION
**Pull request by m6jones**

### CODE REVIEW

File: components/d2l-activity-list-item/d2l-activity-list-item.js [hunk 1]
```diff
@@ -597,7 +604,13 @@ class D2lActivityListItem extends mixinBehaviors([IronResizableBehavior, D2L.Pol
 				this._fetchEntity(imageEntity.href)
 					.then(function(hydratedImageEntity) {
 						this._image = hydratedImageEntity;
-					}.bind(this));
+					}.bind(this))
+					.finally(() => {
+						this.dispatchEvent(new CustomEvent('d2l-activity-image-loaded', {
```
Comment (qwangd2l) [inline components/d2l-activity-list-item/d2l-activity-list-item.js]: I think we need to wait for d2l-course-image to fire its load event before firing this.
Most likely this event? https://github.com/Brightspace/course-image/blob/master/d2l-course-image.js#L170

Comment (m6jones) [reply components/d2l-activity-list-item/d2l-activity-list-item.js]: Oh!! thanks!

**Comment by qwangd2l** [comment]
Hmm I'm having some issues listening to the text-loaded event here. Any ideas?
`d2l-activity-image-loaded` seems to be fine.
```
ready() {
    this.addEventListener('d2l-activity-text-loaded', this._removeTextPlaceholders); // not getting the event
    this.addEventListener('d2l-activity-image-loaded', this._removeImageShimmers); // good
}
```

**Comment by m6jones** [comment]
My only guess is that for some reason it isn't getting to the code that fires it. Since I believe they are the same.

Tests seem to work.

**Comment by qwangd2l** [comment]
> My only guess is that for some reason it isn't getting to the code that fires it. Since I believe they are the same.
> 
> Tests seem to work.

Will investigate further :).

**Pull request merged**